File: JavaScript/kyu-7/_0020__categorize-new-member/solution.js
function openOrSenior(data) {
  return Array.from({ length: data.length }, (_e, i) => (data[i][0] >= 55 && data[i][1] > 7 ? 'Senior' : 'Open'));
}

console.log(
  openOrSenior([
    [18, 20],
    [45, 2],
    [61, 12],
    [37, 6],
    [21, 21],
    [78, 9],
  ]),
);

File: Python/kyu-7/_0020__categorize-new-member/solution.py
def open_or_senior(data):
    return ['Senior' if el[0] >= 55 and el[1] > 7 else 'Open' for el in data]

print(open_or_senior([[18, 20], [45, 2], [61, 12], [37, 6], [21, 21], [78, 9]]))